src/lightkurve/targetpixelfile.py

```diff
@@ -1365,6 +1376,13 @@ def interact(
             >>> transform_func = lambda lc: lc.normalize()  # doctest: +SKIP
             >>> tpf.interact(ylim_func=ylim_func, transform_func=transform_func)  # doctest: +SKIP
 
+        the lightcurve after each pixel selection::
+
+            >>> interact_mask = tpf.interact(return_selection_mask=True)  # doctest: +SKIP
+            >>> # Once the desired pixels have been selected, save the result in
+            >>> # a separate variable to "freeze" the selection.
+            >>> my_custom_mask = interact_mask.copy()  # doctest: +SKIP
```

Comment:
Does this need to be copied to edit interact_mask? I don't think that is the case, so I might skip the my_custom_mask example in the header.

Reply:
`interact_mask.copy()` is not strictly necessary, but is often needed in practice. That's why I include it in the example.

Reason:
`interact_mask` reflects the live / active selection. In practice, the selection can easily be inadvertently changed, surprising users. Scenario:
1. run `interact()` in a cell, and select a custom aperture
2. use the custom aperture (in `interact_mask`) to create a lightcurve.
3. do something else in other cells.
4. try to use the custom aperture again.

In my experience, after step 3 (do something else in other cells), in many cases the selection in `interact()` would be changed (to no pixel selected usually) [*]. In such scenario, if users do not pay attention and use the live `interact_mask`, they would have used a different aperture from what they initially selected.

The step of `my_custom_mask = interact_mask.copy()` in the example is one way to "freeze" / save the custom aperture mask while the users are still conscious of what they have just selected, avoiding any surprise from inadvertent changes.

---
[*] After I use `tpf.interact()`, scroll to other parts of the notebook, and scroll back to `tpf.interact()` cell, in many cases the selection is somehow lost:

![image](https://github.com/lightkurve/lightkurve/assets/250644/3ae2e369-7562-43ee-be09-405f28b08003)